media-plugins/libvisual-plugins/libvisual-plugins-0.4.0-r3.ebuild

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
```

Comment:
As you don't change this file, no need to update the copyright line.

Reply:
Yes, I've thought about this one. However, repoman complained so I changed it